__tests__/ApplicationTest.js

```diff
@@ -57,4 +78,52 @@ describe("자동차 경주", () => {
     // then
     await expect(app.run()).rejects.toThrow("[ERROR]");
   });
+
+  test("예외 테스트 - 공백 자동차 이름", async () => {
+    // given
+    const inputs = ["pobi,,java"];
+    mockQuestions(inputs);
+
+    // when
+    const app = new App();
+
+    // then
+    await expect(app.run()).rejects.toThrow("[ERROR]");
+  });
+
+  test("예외 테스트 - 같은 자동차 이름", async () => {
+    // given
+    const inputs = ["pobi,pobi"];
+    mockQuestions(inputs);
+
+    // when
+    const app = new App();
+
+    // then
+    await expect(app.run()).rejects.toThrow("[ERROR]");
+  });
+
+  test("예외 테스트 - 숫자가 아닌 시도 횟수", async () => {
+    // given
+    const inputs = ["pobi,java", "number"];
+    mockQuestions(inputs);
+
+    // when
+    const app = new App();
+
+    // then
+    await expect(app.run()).rejects.toThrow("[ERROR]");
+  });
+
+  test("예외 테스트 - 음수 시도 횟수", async () => {
+    // given
+    const inputs = ["pobi,java", "-1"];
+    mockQuestions(inputs);
+
+    // when
+    const app = new App();
+
+    // then
+    await expect(app.run()).rejects.toThrow("[ERROR]");
```

Comment:
```suggestion
await expect(app.run()).rejects.toThrow("[ERROR] 시도 횟수를 양수로 입력해 주세요");
```
테스트 할때 이렇게 구체적으로 ERROR문을 적으면 더 정확하게 테스트를 처리할 수 있을 것 같아요 !